tests/test_climate_baseline_helpers.py

```diff
@@ -0,0 +1,169 @@
+from pycds.climate_baseline_helpers import \
+    pcic_climate_variable_network_name, \
+    get_or_create_pcic_climate_variables_network, \
+    create_pcic_climate_baseline_variables, \
+    load_pcic_climate_baseline_values, \
+    field_format, \
+    InvalidVariableName
+
+
+@fixture
+def empty_database_session(mod_blank_postgis_session):
+    sesh = mod_blank_postgis_session
+    engine = sesh.get_bind()
+    pycds.Base.metadata.create_all(bind=engine)
+    yield sesh
+
+
+@fixture
+def sesh_with_climate_baseline_variables(empty_database_session):
+    sesh = empty_database_session
+    create_pcic_climate_baseline_variables(sesh)
+    yield sesh
+
+
+@fixture
+def stations():
+    return [Station(id=1, native_id='100'), Station(id=2, native_id='200'), ]
+
+
+@fixture
+def histories(stations):
+    return [History(
+        station_id=station.id,
+        station_name='Station {0}'.format(station.native_id),
+        sdate=datetime.datetime(year, 1, 1),
+        edate=datetime.datetime(year+1, 1, 1),
+    ) for station in stations for year in [2000, 2001]]
+
+
+def describe_get__or__create__pcic__climate__variables__network():
+
+    def test_creates_the_expected_new_network_record(empty_database_session):
+        sesh = empty_database_session
+        network = get_or_create_pcic_climate_variables_network(sesh)
+        results = sesh.query(Network).filter(Network.name == pcic_climate_variable_network_name)
+        assert results.count() == 1
+        result = results.first()
+        assert network.id == result.id
+        assert result.publish == True
+        assert 'PCIC' in result.long_name
+
+    def test_creates_no_more_than_one_of_them(empty_database_session):
+        sesh = empty_database_session
+        get_or_create_pcic_climate_variables_network(sesh)
+        get_or_create_pcic_climate_variables_network(sesh)
+        results = sesh.query(Network).filter(Network.name == pcic_climate_variable_network_name)
+        assert results.count() == 1
+
+def describe_create__pcic__climate__baseline__variables():
+
+    def test_causes_network_to_be_created(sesh_with_climate_baseline_variables):
+        sesh = sesh_with_climate_baseline_variables
+        results = sesh.query(Network).filter(Network.name == pcic_climate_variable_network_name)
+        assert results.count() == 1
+
+    @mark.parametrize('name, keyword, kwd', [
+        ('Tx_Climatology', 'maximum', 'Max.'),
+        ('Tn_Climatology', 'minimum', 'Min.'),
+    ])
+    def test_creates_temperature_variables(sesh_with_climate_baseline_variables, name, keyword, kwd):
+        sesh = sesh_with_climate_baseline_variables
+        network = get_or_create_pcic_climate_variables_network(sesh)
+        result = sesh.query(Variable).filter(Variable.name == name).first()
+        assert result
+        assert (result.unit, result.standard_name, result.network_id) == \
+               (u'celsius', u'air_temperature', network.id)
```

Comment:
You shouldn't need to use the u'style_strings'. Native string objects should be sufficient. Is there a particular reason that you did so?